app/javascript/__tests__/modules/listingDetails/ListingDetailsPricingTable.test.tsx

```diff
@@ -1,19 +1,51 @@
 import React from "react"
-import renderer from "react-test-renderer"
+import renderer, { act } from "react-test-renderer"
 import { ListingDetailsPricingTable } from "../../../modules/listingDetails/ListingDetailsPricingTable"
 import { closedRentalListing } from "../../data/RailsRentalListing/listing-rental-closed"
-import { habitatListing } from "../../data/RailsSaleListing/listing-sale-habitat"
+// import { habitatListing } from "../../data/RailsSaleListing/listing-sale-habitat"
```

Comment:
I'm always a fan of getting rid of commented code! But this is a little unique as we ended up rolling back some of the AC of the ticket, so extra work was done than what was needed. I added future tickets for the habitat listings, so since we'll work a habitatListing test soon, I vote to keep it in.